packages/localize/testdata/transform/goldens/xliff/es-419.xlf

```diff
@@ -1,23 +1,34 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<xliff version="1.2" xmlns="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-strict.xsd">
-<file target-language="es-419" source-language="en" original="lit-localize-inputs" datatype="plaintext">
-<body>
-<trans-unit id="string">
-  <source>Hello World!</source>
-  <target>Hola Mundo!</target>
-</trans-unit>
-<trans-unit id="lit">
-  <source>Hello <ph id="0">&lt;b>&lt;i></ph>World!<ph id="1">&lt;/i>&lt;/b></ph></source>
-  <target>Hola <ph id="0">&lt;b>&lt;i></ph>Mundo!<ph id="1">&lt;/i>&lt;/b></ph></target>
-</trans-unit>
-<trans-unit id="variables_1">
-  <source>Hello <ph id="0">${name}</ph>!</source>
-  <target>Hola <ph id="0">${name}</ph>!</target>
-</trans-unit>
-<trans-unit id="lit_variables_1">
-  <source>Hello <ph id="0">${name}</ph>, click <ph id="1">&lt;a href="${url}"></ph>here<ph id="2">&lt;/a></ph>!</source>
-  <target>Hola <ph id="0">${name}</ph>, clic <ph id="1">&lt;a href="${url}"></ph>aquí<ph id="2">&lt;/a></ph>!</target>
-</trans-unit>
-</body>
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Why the change in header? This is the reverse of what happened in one file in the natural msg expressions PR

Reply:
Hm, I think something weird happened. But it's fixed now. This PR wasn't based on the natural expressions one originally, but now it is.

I also got really confused about which tests were which while looking into this, so I've renamed them all to be more clear.